src/frame/mod.rs

```diff
@@ -1,35 +1,34 @@
 use std::{mem, slice};
 
-use apple_sys::{CoreMedia::{CFDictionaryGetValue, CFDictionaryRef, CFTypeRef, CFNumberGetValue, CFNumberType_kCFNumberSInt64Type}, ScreenCaptureKit::{SCStreamFrameInfoStatus, SCFrameStatus_SCFrameStatusComplete}};
+use apple_sys::{
+    CoreMedia::{
+        CFDictionaryGetValue, CFDictionaryRef, CFNumberGetValue, CFNumberType_kCFNumberSInt64Type,
+        CFTypeRef,
+    },
+    ScreenCaptureKit::{SCFrameStatus_SCFrameStatusComplete, SCStreamFrameInfoStatus},
+};
+use core_graphics::{
+    access::ScreenCaptureAccess,
+    display::{CFArrayGetCount, CFArrayGetValueAtIndex, CFArrayRef, CGDirectDisplayID, CGDisplay},
+};
+use core_video_sys::{
+    CVPixelBufferGetBaseAddressOfPlane, CVPixelBufferGetBytesPerRowOfPlane, CVPixelBufferGetHeight,
+    CVPixelBufferGetWidth, CVPixelBufferLockBaseAddress, CVPixelBufferRef,
+    CVPixelBufferUnlockBaseAddress,
+};
 use screencapturekit::{
     sc_content_filter::{InitParams, SCContentFilter},
+    sc_display::SCDisplay,
     sc_error_handler::StreamErrorHandler,
     sc_output_handler::{CMSampleBuffer, SCStreamOutputType, StreamOutput},
     sc_shareable_content::SCShareableContent,
     sc_stream::SCStream,
     sc_stream_configuration::SCStreamConfiguration,
-    sc_sys::{SCFrameStatus, CMSampleBufferGetPresentationTimeStamp, CMSampleBufferGetImageBuffer, CMSampleBufferGetSampleAttachmentsArray, }, sc_display::SCDisplay,
-};
-use core_graphics::{
-    access::ScreenCaptureAccess,
-    display::{CGDirectDisplayID, CGDisplay, CFArrayGetCount, CFArrayGetValueAtIndex, CFArrayRef, },
+    sc_sys::{
+        CMSampleBufferGetImageBuffer, CMSampleBufferGetPresentationTimeStamp,
+        CMSampleBufferGetSampleAttachmentsArray, SCFrameStatus,
+    },
 };
-use core_video_sys::{CVPixelBufferRef, CVPixelBufferLockBaseAddress, CVPixelBufferGetWidth, CVPixelBufferGetHeight, CVPixelBufferGetBaseAddressOfPlane, CVPixelBufferGetBytesPerRowOfPlane, CVPixelBufferUnlockBaseAddress};
-
-pub struct YUVFrame {
```

Comment:
Is there a reason for moving this from here to `lib.rs` ?

Reply:
Yea frame.rs is filled with macos only code but this struct is used in other places.